components/interactive-markdown.js

```diff
@@ -23,41 +29,101 @@ function ClickToRenderCodePreview(props) {
   )
 }
 
-function interactiveMarkdown(markdownString) {
-  const componentBlocks = []
+function highlightWithPrism(code, language) {
+  return Prism.highlight(code, Prism.languages[language])
+}
+
+function createSupportedPragmaMatcher() {
+  const prismHighlighter = highlighter(highlightWithPrism)
```

Comment:
Please see https://github.com/kentcdodds/glamorous-website/pull/125/files#diff-d9315b560685be6f1eba31b8e3484801R3 for more information and thoughts on how the handlers are created and setup.